examples/pdc_resource_tests.rb

```diff
@@ -172,6 +172,19 @@ def token
   end
 end
 
+describe PDC::V1::ContentDeliveryRepo do
+  let(:repo_class) { PDC::V1::ContentDeliveryRepo }
+  let(:existing_repo) { repo_class.all!.first }
+
+  describe '#find' do
```

Comment:
Need a test for this class instead of an example so that we know if something breaks.